playbooks/Splunk_Attack_Analyzer_Dynamic_Analysis.py

```diff
@@ -508,10 +510,12 @@ def normalized_file_forensic_output(action=None, success=None, container=None, r
     
     ## pair forensic job results with url detonated
     job_file_dict = {}
-    for orig_url, orig_job, filtered_job in zip(filtered_result_0_parameter_file, filtered_result_0_data___jobid, filtered_result_1_parameter_job_id):
-        if orig_job == filtered_job:
-            job_file_dict[filtered_job] = orig_url
 
+    for orig_url, orig_job in zip(filtered_result_0_parameter_file, filtered_result_0_data___jobid):
```

Comment:
@tccontre This references url but also file. Same with the comments for this block. Should this reference a file and not a url?

Reply:
sorry @kelby-shelton I missed this feedback. minor fixed done